### PR TITLE
Adding initial Landr content

### PR DESCRIPTION
**Pull request by andrewnhem**
Change-type: minor

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,15 +1,114 @@
...
+### Commit examples
+
+Here are some examples of valid commits:
+
+**Big new feature**
+
+```
+multi-room: Add multi-room feature
+
+This feature adds multi-room audio streaming to balenaSound.
```
Comment (iamsolankiamit) [inline CONTRIBUTING.md:53]: let's update this for balenaSense

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,15 +1,114 @@
...
+Change-type: patch
+Connects-to: #90
+```
+
+## PR guidelines
+Pull requests are the only way of pushing your code to the `master` branch. When creating a PR make sure you choose a short but sensical PR title and add a few lines describing your contribution.
+
+### PR approval
+The PR will only be able to be merged only after:
+- It has been approved at least by one [codeowner](https://github.com/balenalabs/balena-sound/blob/master/.github/CODEOWNERS)
```
Comment (iamsolankiamit) [inline CONTRIBUTING.md:90]: We are moving away from codeowners right!

Comment (andrewnhem) [reply CONTRIBUTING.md:90]: Good catch!

File: CONTRIBUTING.md (hunk 1 above)
Comment (andrewnhem) [reply CONTRIBUTING.md:53]: +1

**Comment by andrewnhem** [comment]
@iamsolankiamit @cmfcruz If this is looking close, let's get it out there and redirect that subdomain to it plz :)

Can always hotfix more as we get more user feedback.

**Review by iamsolankiamit — APPROVED** [review]

**Pull request merged**